<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pomodoro_Settings_Window</class>
 <widget class="QMainWindow" name="Pomodoro_Settings_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>745</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pomodoro Settings</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="combo_box_type_of_pomodoro">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>110</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>681</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Type of Pomodoro</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>210</y>
      <width>781</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Choose Number of Sessions</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin_box_number_of_sessions">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>300</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_wrong_inputs">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>650</y>
      <width>611</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>499</y>
      <width>261</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="btn_cancel">
       <property name="font">
        <font>
         <family>MS Sans Serif</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border-style: solid;
	border-width: 2px;
	border-color: #9999aa;
	border-radius: 10px;
	color: white;
	background-color: #3474eb
}

QPushButton:enabled {
	background-color: #3474eb
	color: white;
}

QPushButton:pressed {
	background-color: #0d2f72;
	color: #fffffe;
}

QPushButton:hover:!pressed {
	background-color: E1F4FF;
	color: 0c2f70;
}

QPushButton:disabled {
	background-color: #aaaaaa;
	color: #ffffff;
}</string>
       </property>
       <property name="text">
        <string>Cancel Changes</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>500</y>
      <width>271</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="btn_save">
       <property name="font">
        <font>
         <family>MS Sans Serif</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border-style: solid;
	border-width: 2px;
	border-color: #9999aa;
	border-radius: 10px;
	color: white;
	background-color: #3474eb
}

QPushButton:enabled {
	background-color: #3474eb
	color: white;
}

QPushButton:pressed {
	background-color: #0d2f72;
	color: #fffffe;
}

QPushButton:hover:!pressed {
	background-color: E1F4FF;
	color: 0c2f70;
}

QPushButton:disabled {
	background-color: #aaaaaa;
	color: #ffffff;
}</string>
       </property>
       <property name="text">
        <string>Save Changes</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>852</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
